packages/ui/src/app/pages/Profile/components/AccountItem.tsx

```diff
@@ -36,7 +38,9 @@ export const AccountItem = ({ account }: AccountItemDataProps) => {
           <DropDownButton onClick={setDropped} isDropped={isDropped} size="medium" />
         </AccountControls>
       </AccountItemWrap>
-      <DropDownToggle isDropped={isDropped}>I'm opened</DropDownToggle>
+      <DropDownToggle isDropped={isDropped}>
+        <BalanceDetails locks={balance?.locks || []} recoverables={[]} />
+      </DropDownToggle>
```

Comment:
Just because we have proper slices of the UI part on the way please revert the changes for the opened part.